src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Server/Startup.cs

```diff
@@ -26,8 +19,10 @@
 using Microsoft.Extensions.Hosting;
 using System.Linq;
 #if (IndividualLocalAuth)
-using ComponentsWebAssembly_CSharp.Server.Data;
-using ComponentsWebAssembly_CSharp.Server.Models;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
```

Comment:
Curious why the changes under IndividualLocalAuth was made in https://github.com/AzureAD/microsoft-identity-web/blob/master/ProjectTemplates/templates/ComponentsWebAssembly-CSharp/Server/Startup.cs#L22

Reply:
This is to prepare the .AddAuthentication() pattern that Barry wants;

Reply:
Is that something we need now or is it part of api improvements we're doing later?

Reply:
Yes, it is.